src/main/java/uk/gov/hmcts/reform/jobscheduler/ResponseExceptionHandler.java

```diff
@@ -23,6 +30,24 @@ protected void activateDirectFieldAccess(DataBinder dataBinder) {
         dataBinder.initDirectFieldAccess();
     }
 
+    @Override
+    protected ResponseEntity<Object> handleMethodArgumentNotValid(
```

Comment:
We need to add tests verifying the response. This also gives us a good opportunity to cover all validated fields and check if the response mentions each of the invalid ones.